packages/main/src/themes/Link.css

```diff
@@ -71,11 +71,38 @@
 	color: inherit;
 }
 
+/* For cases when the text wraps, the icon should be displayed at the last row */
+:host([has-end-icon]) .ui5-link-root {
+	display: inline-flex;
+	align-items: end;
+}
+
 :host .ui5-link-root {
 	outline-offset: -0.0625rem;
 	border-radius: var(--_ui5_link_focus_border-radius);
 }
 
+.ui5-link-icon, 
+.ui5-link-end-icon {
+	color: inherit;
+	flex-shrink: 0;
+}
+
+.ui5-link-icon {
+	float: inline-start;
+	margin-inline-end: 0.125rem;
+}
+
+.ui5-link-end-icon {
+	float: inline-end;
+	margin-inline-start: 0.125rem;
+}
+
+.ui5-link-text {
+	overflow: hidden;
```

Comment:
This should be aligned with the change for making wrapping of the Link text by default. The code is related to the topic, so this should be considered in the other PR and adjust it if necessary, and vice versa.

Reply:
Yes, depending on which one will get merged first, I will adjust the PR accordingly. Thank you